net-im/dino/dino-9999.ebuild

```diff
@@ -32,6 +32,7 @@ RDEPEND="
 	x11-libs/gdk-pixbuf:2
 	x11-libs/gtk+:3
 	x11-libs/pango
+	net-libs/libsignal-protocol-c
```

Comment:
I have sorted the list (to the extent possible).

Also, not specifying the atom slot operator `0=` is a bit risky.
In case of an update to an ABI-incompatible new version of libsignal-protocol-c (which warrants a slot update), a `0=` here would automatically trigger a rebuild of `dino`.